src/platforms/apple/common/usage.mdx

```diff
@@ -50,6 +50,55 @@ This integration is the core part of the SDK. It hooks into all signal and excep
 
 This integration will swizzle some methods to create breadcrumbs e.g.: for `UIApplicationDidReceiveMemoryWarningNotification`, `sendAction:to:from:forEvent:` (UI interactions) or `viewDidAppear:` those breadcrumbs will be attached to your events.
 
+## In App Stack Frames
+
+To highlight essential frames of your stacktrace, the Sentry UI only displays frames related to your application by default. Sentry marks such frames as inApp.
+To achieve this, the SDK only sets the inApp flag for frames originating from the main executable by using the [CFBundleExecutable].
+Public frameworks such as UIKitCore, CoreFoundation, GraphicsServices, etc., are always marked as not inApp.
+For private frameworks, such as Sentry, there is a difference between dynamic or static frameworks. If you are not familiar with these terms, we recommend
+reading [Dynamic Library Programming Topics].
+
+[CFBundleExecutable]: https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleexecutable
+[Dynamic Library Programming Topics]: https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/DynamicLibraries/000-Introduction/Introduction.html
+
+### Dynamic Frameworks
+If you use dynamic frameworks, such as, for example, Sentry, the SDK marks these as not inApp. In case you have a private framework that should be inApp,
```

Comment:
```suggestion
If you use dynamic frameworks, such as, for example, Sentry, the SDK marks these as `not inApp`. In case you have a private framework that should be `inApp`,
```